Spring.FluentContext.Web/Spring.FluentContext.Web.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectGuid>{9FC6E664-D984-4D1F-B637-79709F958CA2}</ProjectGuid>
+    <TargetFramework>net461</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugType>full</DebugType>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="D3S.Spring.Web" Version="3.0.0-pre" />
```

Comment:
This should be referenced for netstandard. NetFramework should refer to default Spring packages like in `Spring.FluentContext` project.

Reply:
Changed to default, but I will have to keep my version in separate branch. This project can't target netstandard as it references `System.Web`.